language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -794,12 +794,16 @@ object DottyLanguageServer {
 
     val label = sym.name.show
     val item = new lsp4j.CompletionItem(label)
-    item.setDetail(sym.info.widenTermRefExpr.show)
+    val detail = if (sym.isType) sym.showFullName else sym.info.widenTermRefExpr.show
+    item.setDetail(detail)
+    ParsedComment.docOf(sym).foreach { doc =>
+      item.setDocumentation(markupContent(doc.renderAsMarkdown))
+    }
```

Comment:
While you're extending what we show in completion: CompletionItem has a `deprecated` field we could set now.

Reply:
Good idea. That's done, but I can't see a different in VSCode. The tests show it is actually marked as deprecated, though...